### DISCUSSION TITLE
🚀  更新清單請求 zuvio.tw zuvio.com.tw

### DISCUSSION THREAD

**Discussion by c2xusnpq6** [FutaGuard/LowTechFilter#49] [Q&A]
## 現在 `FutaFilter` 對他的處置是？
嘸

## 建議對此網域做出何種改變
新增

## 提出更改的理由
tracker
https://irs.zuvio.com.tw/student5/chickenM/article/164440 --> https://forum.zuvio.com.tw/api/article/finish

```
forum.zuvio.com.tw/api/article/finish
irs.zuvio.com.tw/api/article/finish (not sure)
```

**Comment by c2xusnpq6** [comment]
mistake............ pls ignore this

https://qt.zuvio.com.tw/student5/chickenM/article/2854 --> https://forumqt.zuvio.com.tw/api/article/finish
```
forumqt.zuvio.com.tw/api/article/finish
```
![image](https://user-images.githubusercontent.com/75058881/114733826-4092db00-9d76-11eb-9241-d48965fcef17.png)

**Comment by tasi788** [comment]
上面所提供的資訊或圖片沒有顯示他是 tracker 的證據，請補充。
